synapse/util/caches/dictionary_cache.py

```diff
@@ -145,21 +170,21 @@ def get(
         Returns:
             DictionaryEntry
```

Comment:
this doesn't document what happens if `dict_keys` is None, but we don't have the full dict cached. Could it, please?

Reply:
... please?

I think it returns some fixed shape of DictionaryEntry ?

Reply:
Ah, I'd put some docs under the `dict_keys` arg (and there is some in `DictionaryEntry`), but have also added some in the return value too.